src/scripts/modules/add_destination.js

```diff
@@ -55,24 +54,22 @@ function updateFormState(fieldName, value) {
   formState[fieldName] = value;
 }
 
-// function storeNew() {
-//   storedCity = localStorage.setItem(
-//     "newDestination",
-//     JSON.stringify(formState)
-//   );
-// }
+function storeNew() {
+  localStorage.setItem("newDestination", JSON.stringify(formState));
+}
 
 // localStorage.clear();
 
-// function loadNew() {
-//   let isStored = localStorage.getItem("newDestination");
-//   if (isStored) {
-//     let data = JSON.parse(isStored);
-//   } else {
-//     data = [];
-//   }
-//   localStorage.setItem("newDestination", data);
-// }
+function loadNew() {
+  let isStored = localStorage.getItem("newDestination");
+  let data;
+  if (isStored) {
+    data = JSON.parse(isStored);
+  } else {
+    data = [];
+  }
+  localStorage.setItem("newDestination", data);
```

Comment:
A LS item must be a string, always. In this case we need to [stringify](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) an object/array before we store it.